app/utils/pages.tsx

```diff
@@ -10,26 +13,36 @@ interface IPage {
     path: string
 }
 
+export function getPathsToMap (){
+    const pathToLeagueData = path.join(process.cwd(), "app", "leagueConfiguration");
+    const pathsToMap = fs.readdirSync(pathToLeagueData);
+    return pathsToMap;
+}
+
+export function getConfigurationPage (filename:string){
```

Comment:
Why did you call this function this? Is this ever going to be a page?

Reply:
Changed name